operator/constants/version.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+var (
+	ver = "0.0.0"
```

Comment:
This is a good pattern for injecting build-time variables, while avoiding source code changes and avoiding imprecisions (should someone forget to update this) 

https://blog.alexellis.io/inject-build-time-vars-golang/